src/schema-generation/order-by-and-pagination-augmentation.ts

```diff
@@ -50,7 +64,11 @@ export class OrderByAndPaginationAugmentation {
         private readonly rootFieldHelper: RootFieldHelper,
     ) {}
 
-    augment(schemaField: QueryNodeField, type: Type): QueryNodeField {
+    augment(
+        schemaField: QueryNodeField,
+        type: Type,
+        options?: OrderByAndPaginationAugmentationOptions,
```

Comment:
```suggestion
        options: OrderByAndPaginationAugmentationOptions = {},
```

this makes sure skipping the options is the same as not specifying any options, and it saves you the `?.` further down which can sometimes be hard to understand